src/native/corehost/ijwhost/ijwhost.cpp

```diff
@@ -39,8 +40,18 @@ pal::hresult_t get_load_in_memory_assembly_delegate(pal::dll_t handle, load_in_m
 
             return StatusCode::Success;
         },
-        delegate
+        &get_function_pointer
     );
+    if (status != StatusCode::Success)
+        return status;
+
+    return get_function_pointer(
+        _X("Internal.Runtime.InteropServices.InMemoryAssemblyLoader"),
```

Comment:
I think it would be better to use assembly qualified name here - in theory some other assembly can define the same type and I honestly don't know what `Type.GetType` will do - probably return the first match in undefined order.
Note that the existing `hdt` constants are all translated to assembly qualified names (see hostpolicy.cpp:490).

Reply:
Updated.